test/functional/developer_portal/admin/messages/outbox_controller_test.rb

```diff
@@ -21,7 +23,7 @@ def setup
   test "creates messages with origin == 'web'" do
     buyer = FactoryBot.create :buyer_account, :provider_account => @provider
 
-    post :create, :message => { :subject => "message via web", :body => "message via web" }, :to => buyer.id
+    post :create, params: { message: { :subject => "message via web", :body => "message via web" }, :to => buyer.id }
```

Comment:
ideally change all to `key: value` format but can be done in a future PR
same applies to numerous other places

Reply:
@akostadinov Yes we will do it folder wise for the application there are lot of places need to updated.